README.md

```diff
@@ -43,16 +42,18 @@ Important: when after saving an error appears in the Splunk Web tool, the config
 it does not give much information what went wrong.
 To find out the issue, you will have to look in the log file (see below).
 
-An API or Machine Token is required to access the Threat Intelligence Engine or TIE. Both the legacy
-token created through `tie.dcso.de` and the newer tokens created through `portal.dcso.de` are supported.
-If you have any questions about this Token, please contact DCSO (see below).
+DCSO API credentials are required to access the Threat Intelligence Engine or TIE. The DCSO API credetials 
+can be retrieved on *https://developer.dcso.de/*. 
+If you have any questions about the credentials, please contact DCSO (see below).
 
 There are few more details about the configuration:
 
-* **API Token**: either a legacy tie.dcso.de token, or new one created through the DCSO Portal.
-* **Initial IoC Sequence Number**: Use this to start from a particular sequence number. This is
-  useful when re-installing or upgrading to incompatible add-on version (data in index would
-  stay compatible). Leave 0 to use whatever number is stored or start from NOW minus 30 days. 
+* **API Client ID**: DCSO api client_id from the DCSO Developer Portal.
```

Comment:
is developer portal the only way to get credentials? i would not assume so. maybe reference idm credentials instead

Reply:
good point